project/src/main/java/com/google/sps/data/ValidationUtils.java

```diff
@@ -29,6 +33,10 @@
     private static final int MAX_PRICE_LEVEL = 4;
     /** The minimal valid price level value. */
     private static final int MIN_PRICE_LEVEL = 0;
+    /** The optional business statuses. */
+    private static final List<String> OPTIONAL_STATUS =
```

Comment:
I was confused by the name, I think maybe something like "valid statuses" would be more clear at least for me.

Reply:
You are right, valid is much better.